raiden/log_config.py

```diff
@@ -166,21 +165,36 @@ def configure_logging(
     })
     _wrap_tracebackexception_format(redact)
 
-    enabled_log_handlers = []
+    handlers = dict()
     if log_file:
-        enabled_log_handlers.append('file')
+        handlers['file'] = {
+            'class': 'logging.handlers.WatchedFileHandler',
+            'filename': log_file,
```

Comment:
The `None` check is gone here for `log_file`, will that be a problem?

Reply:
I don't think so. If `log_file is None`, the handler is not configured, so the log_file variable won't be used at all.

I assume the previous code just used the `else` part as dirty way to turn off the log file creation. Now that we only add the handler when we want the log file, this should be unnecessary.

Reply:
Ah right, my bad.